numpy/_core/tests/examples/limited_api/meson.build

```diff
@@ -34,6 +34,16 @@ py.extension_module(
     limited_api: '3.6',
 )
 
+py.extension_module(
+    'limited_api_latest',
+    'limited_api_latest.c',
+    c_args: [
+      '-DNPY_NO_DEPRECATED_API=NPY_1_21_API_VERSION',
```

Comment:
This makes me wonder if a macro that corresponds to "the latest numpy C API version" would be useful. If it doesn't exist already.